lit/docs/auth/managing-teams.lit

```diff
@@ -76,6 +76,28 @@
   }
 }
 
+\section{
+  \title{\code{fly active-users}}{fly-active-users}
+
+  To list all users that have logged into your instance in the last two months,
+  run:
+
+  \codeblock{bash}{{
+  $ fly -t example active-users
+  }}
+
+  The output will include the username, connector (which method they used to
+  authenticate) and the date of their last login.
+
+  You can list users whose last login was within a different range by using:
+
+  \codeblock{bash}{{
+  $ fly -t example active-users --since yyyy-MM-dd
+  }}
+
+  This can be helpful to get a sense of how active your cluster is. 
+}
+
```

Comment:
Non-blocking question: do we have a standard format for placeholders in examples?

Some docs sites use:
```
$ fly -t <example> active-users --since <date in yyyy-MM-dd format>
```

Pivotal docs seem to use:
```
$ fly -t EXAMPLE active-users --since DATE
```
    Where:
    * `Example` is the target Concourse
    * `DATE` is the date to search from, written in `yyyy-MM-dd` format

Reply:
I was just kinda following my nose on this one, and I definitely have a subjective preference for the former.

Either way, I'd love to have such a standard -- though I'm not sure how to enforce it. Feels like a good moment to loop @vito in for the BDFL context.